cnxdb/init/main.py

```diff
@@ -45,7 +45,15 @@ def _has_schema(cursor):
 
 
 def init_db(engine, as_venv_importable=False):
-    """Initialize the database from the given ``connection_string``."""
+    """Initialize the database from the given ``connection_string``.
```

Comment:
`s/connection_string/db connection engine/` no?

Reply:
Addressed in #99